src/components/AutoComplete/AutoComplete.tsx

```diff
@@ -56,6 +56,7 @@ function AutoComplete<Item extends ListItemKey = ListItemKey>(
     __internal__highlightedIndex,
     __internal__loading,
     __internal__focus,
+    __internal__items = [], // purely for Kitchen sink
```

Comment:
Let's remove the comment. Otherwise, we should probably have it for all `__internal__` props.